React/Fabric/RCTSurfaceTouchHandler.mm

```diff
@@ -80,14 +79,19 @@ bool operator()(const ActiveTouch &lhs, const ActiveTouch &rhs) const
 
 static void UpdateActiveTouchWithUITouch(
     ActiveTouch &activeTouch,
-    RCTUITouch *uiTouch, // [macOS]
+    RCTUITouch *uiTouch,
     RCTUIView *rootComponentView,
     CGPoint rootViewOriginOffset) // [macOS]
 {
 #if !TARGET_OS_OSX // [macOS]
   CGPoint offsetPoint = [uiTouch locationInView:activeTouch.componentView];
   CGPoint screenPoint = [uiTouch locationInView:uiTouch.window];
   CGPoint pagePoint = [uiTouch locationInView:rootComponentView];
+#else // [macOS
+  CGPoint offsetPoint = [activeTouch.componentView convertPoint:uiTouch.locationInWindow fromView:nil];
+  CGPoint screenPoint = uiTouch.locationInWindow;
+  CGPoint pagePoint = CGPointMake(screenPoint.x, CGRectGetHeight(rootComponentView.window.frame) - screenPoint.y);
```

Comment:
I thought that our RCTUIViews overwrote isFlipped so we don't have to do this conversion?

https://developer.apple.com/documentation/appkit/nsview/1483532-flipped?language=objc

Reply:
These are there because RCTUITouch is an NSEvent on macOS, which does not have the locationInView: method used by the iOS version.

Reply:
We could add the missing methods to RCTUITouch on macOS to get rid of this conditional code. I'll test this out and submit a separate PR.